test/unit/calculators/coronavirus_find_support_calculator_test.rb

```diff
@@ -0,0 +1,534 @@
+require_relative "../../test_helper"
+
+module SmartAnswer::Calculators
+  class CoronavirusFindSupportCalculatorTest < ActiveSupport::TestCase
+    setup do
+      @calculator = CoronavirusFindSupportCalculator.new
+    end
+
+    context "#needs_help_with?" do
```

Comment:
Pedantic perhaps, but the content of this block described the behaviour of the `needs_help_with?` instance method of the class. So I think these blocks should be `describe` rather than `context`. So:
```ruby
describe "#needs_help_with?" do
```
The same for the other instance method blocks.

Reply:
Thanks @reggieb - I agree, it should be `describe`.  However, `describe` throws an error!